six/modules/c++/samples/test_image_to_scene.cpp

```diff
@@ -82,44 +82,44 @@ int main(int argc, char** argv)
         six::Container* container = reader.getContainer();
         std::auto_ptr<scene::ProjectionModel> projection;
         std::auto_ptr<scene::SceneGeometry> geom;
+        scene::Vector3 groundPlaneNormal;
         types::RowCol<double> imagePt;
         if (container->getDataType() == six::DataType::COMPLEX)
         {
             const six::sicd::ComplexData* data =
                     reinterpret_cast<six::sicd::ComplexData*>(
                             container->getData(0));
-
             geom.reset(six::sicd::Utilities::getSceneGeometry(data));
             projection.reset(six::sicd::Utilities::getProjectionModel(
                     data, geom.get()));
             imagePt = data->pixelToImagePoint(pixelLoc);
+            groundPlaneNormal = six::sicd::Utilities::getGroundPlaneNormal(*data);
         }
         else if (container->getDataType() == six::DataType::DERIVED)
         {
             const six::sidd::DerivedData* data =
                     reinterpret_cast<six::sidd::DerivedData*>(
                             container->getData(0));
-            projection = six::sidd::Utilities::getProjectionModel(data);
             geom = six::sidd::Utilities::getSceneGeometry(data);
+            projection = six::sidd::Utilities::getProjectionModel(data);
             std::cout << geom->getReferencePosition() << std::endl;
```

Comment:
This is an old `cout` statement left in by mistake back in the day... go ahead and remove